<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seller_newShipmentLog</class>
 <widget class="QDialog" name="Seller_newShipmentLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_goodsName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货品名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsID">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货品编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_storageNum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_place">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_producer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_storageNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>库存量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_place">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>存放地：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_producer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>生产商：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_shipmentNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>出货量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_shipmentNum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_goodsID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
